{{cookiecutter.repo_name}}/docs/conf.py

```diff
@@ -16,15 +16,36 @@
 from __future__ import absolute_import, unicode_literals
 
 import os
+import re
 import sys
 from subprocess import check_call
 
 import edx_theme
 
 import django
+from django.conf import settings
 from django.utils import six
 
+
+def get_version(*file_paths):
+    """
+    Extract the version string from the file at the given relative path fragments.
+    """
+    filename = os.path.join(os.path.dirname(__file__), *file_paths)
+    version_file = open(filename).read()    # pylint: disable=open-builtin
+    version_match = re.search(r"^__version__ = ['\"]([^'\"]*)['\"]", version_file, re.M)
+    if version_match:
+        return version_match.group(1)
+    raise RuntimeError('Unable to find version string.')
+
+
+REPO_ROOT = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+sys.path.append(REPO_ROOT)
+
+VERSION = get_version('../3{{ cookiecutter.app_name }}', '__init__.py')
```

Comment:
@jmbowman yes, thank you for catching this.

In case you're interested how did it even happen - I'm on Ubuntu with PC keyboard (with Win key); Win+Number launches/switches to the app at "Number" position in the dash (similar to what "Snap" app does on Mac). On my system Win+3 is terminal - probably I've released Win key too fast (it happens to me sometimes) and didn't notice.